apps/dashboard/app/javascript/turbo_shim.js

```diff
@@ -5,21 +5,26 @@
   this shim until we enable it.
 */
 
-import { setInnerHTML } from './utils';
+import { setInnerHTML, setFocus } from './utils';
 import { alert } from './alert';
 
 export function replaceHTML(id, html) {
   const ele = document.getElementById(id);
 
-  if(ele == null){
-    return;
-  } else {
-    var tmp = document.createElement('div');
-    tmp.innerHTML = html;
-    const newHTML = tmp.querySelector('template').innerHTML;
-    tmp.remove();
+  if(ele == null) return;
 
-    setInnerHTML(ele, newHTML);
+  var tmp = document.createElement('div');
+  tmp.innerHTML = html;
+  const newHTML = tmp.querySelector('template').innerHTML;
+  tmp.remove();
```

Comment:
Can you try to avoid formatting changes like this? It makes it harder for me to distinguish what's actually changing and what's just been formatted differently.